torch/_decomp/decompositions.py

```diff
@@ -4211,6 +4211,28 @@ def floor_divide(self, other):
     return torch.div(self, other, rounding_mode="floor")
 
 
+@register_decomposition(aten.sum.default)
+def sum_default(
+    self: Tensor,
```

Comment:
How is it possible to reach this overload? It seems like it's not exposed to Python:
```python
>>> torch.randn(3,3).sum(True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sum() received an invalid combination of arguments - got (bool), but expected one of:
 * (*, torch.dtype dtype)
 * (tuple of ints dim, bool keepdim, *, torch.dtype dtype)
 * (tuple of names dim, bool keepdim, *, torch.dtype dtype)
```

Reply:
Ah, good point. I will update the signature of this function. I saw `keepdim` as an argument and assumed that it could be a valid combination with `dtype` 😅